<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MissionWidget</class>
 <widget class="QWidget" name="MissionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="le_nom">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="placeholderText">
          <string>--</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_nouveau">
         <property name="toolTip">
          <string>Ajouter une nouvelle mission</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_ouvrir">
         <property name="toolTip">
          <string>Ouvrir une mission sauvegarder</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_sauvegarder">
         <property name="toolTip">
          <string>Enregistrer la mission</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_supprimer">
         <property name="toolTip">
          <string>Supprimer la mission</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1" colspan="2">
        <widget class="QDateTimeEdit" name="tim_Date">
         <property name="wrapping">
          <bool>true</bool>
         </property>
         <property name="displayFormat">
          <string>HH:mm dd/MM/yyyy</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QDoubleSpinBox" name="dsp_dep_lon">
         <property name="wrapping">
          <bool>true</bool>
         </property>
         <property name="decimals">
          <number>6</number>
         </property>
         <property name="minimum">
          <double>-180.000000000000000</double>
         </property>
         <property name="maximum">
          <double>180.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lb_Traj">
         <property name="text">
          <string>Trajectoire : </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lb_charge">
         <property name="text">
          <string>Charge : </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="dsp_cib_lat">
         <property name="wrapping">
          <bool>true</bool>
         </property>
         <property name="decimals">
          <number>6</number>
         </property>
         <property name="minimum">
          <double>-85.000000000000000</double>
         </property>
         <property name="maximum">
          <double>85.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lb_depart">
         <property name="text">
          <string>DÃ©part : </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QDoubleSpinBox" name="dsp_charge">
         <property name="suffix">
          <string> kg</string>
         </property>
         <property name="maximum">
          <double>10000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Date : </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QDoubleSpinBox" name="dsp_cib_lon">
         <property name="wrapping">
          <bool>true</bool>
         </property>
         <property name="decimals">
          <number>6</number>
         </property>
         <property name="minimum">
          <double>-180.000000000000000</double>
         </property>
         <property name="maximum">
          <double>180.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="pb_Traj_calculer">
           <property name="toolTip">
            <string>Calcul de la trajectoire</string>
           </property>
           <property name="text">
            <string>Calcul</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_Traj_sauvegarder">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Sauvegarder la trajectoire</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lb_cible">
         <property name="text">
          <string>Cible : </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="dsp_dep_lat">
         <property name="wrapping">
          <bool>true</bool>
         </property>
         <property name="decimals">
          <number>6</number>
         </property>
         <property name="minimum">
          <double>-85.000000000000000</double>
         </property>
         <property name="maximum">
          <double>85.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="lbFaisTitle">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
         </property>
        </widget>
       </item>
       <item row="6" column="1" colspan="2">
        <widget class="QLabel" name="lbFaisInfos">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="DroneDetails" name="droneWid" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DroneDetails</class>
   <extends>QWidget</extends>
   <header>GestionDrones/DroneDetails.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
